todoComments/todoComments.md

```diff
@@ -0,0 +1,185 @@
+| 28 | Error Component needs to be implemented, Its currently a bare bones placeholder | src/client/app/components/RouteComponent.tsx | No Issue Needed | - | This looks like a duplicate TODO to make a better ErrorComponent. There is already a TODO in src/client/app/components/router/ErrorComponent.tsx. This TODO should be able to be removed |
+| 29 | Styling for the component, may need to be converted into .css files | src/client/app/components/ThreeDPillComponent.tsx | Related To An Issue | [Github issue #1203](https://github.com/OpenEnergyDashboard/OED/issues/1203) Will decide how CSS is implemented across all pages | This should be decided in Issue 1203.  |
+| 30 | ISSUE when many meters selected they are cut off. | src/client/app/components/ThreeDPillComponent.tsx | Needs Issue Created | - | Pill Components go below the container and out of view when many meters are selected and the width of the screen is below a certain point. Allowing vertical scroll here could be a potential fix. |
+| 31 | Provide default link when there are issues fetching version number | src/client/app/components/TooltipHelpComponent.tsx | Needs Issue Created | - | If there is a problem with fetching the OED version, Tooltip links will display "..." in place of the <a> tag link. A default link could be used instead of the "..." |
+| 32 | This is trying to fix the fact that when the rate menu appears and disappears and you don't go to a new page then the help pop up does not occur. This works reasonably well but you have to click the help icon twice to see the pop up in this case. A better fix would be desirable in the long term since there have been issues with tooltips in multiple places. | src/client/app/components/TooltipMarkerComponent.tsx | Related To An Issue | [Github issue #988](https://github.com/OpenEnergyDashboard/OED/issues/988)  | Issue 988 react-tooltip upgrade from v4 to v5 may solve this problem |
+| 33 | migrate ReactRouter v6 & hooks | src/client/app/components/UnsavedWarningComponent.tsx | Unsure | - | It looks like react-router has already been upgraded to v6. Unsure if there are specific hooks that were in mind that still need to be implemented but this seems okay. React-router does suggest not blocking users requests and to try to use a different method |
+| 34 | translate me | src/client/app/components/UnsavedWarningComponent.tsx | No Issue Needed | - | Looks like this translation has already been added. I believe this TODO comment can be removed |
```

Comment:
I think you are correct that these are both obsolete. You are welcome to do a PR to fix if you want :-)

Reply:
This is addressed in this PR https://github.com/OpenEnergyDashboard/OED/pull/1389